snapcraft/common.py

```diff
@@ -58,8 +62,9 @@ def run_output(cmd, **kwargs):
         f.write('\n')
         f.write('exec $*')
         f.flush()
-        return subprocess.check_output(['/bin/sh', f.name] + cmd,
-                                       **kwargs).decode('utf8').strip()
+        return subprocess.check_output([
+            '/bin/sh', f.name] + cmd, **kwargs).decode(
+                sys.getfilesystemencoding()).strip()
```

Comment:
This changes the signature of `run_output` from returning a string to returning bytes. This on its own won't raise an exception, but the caller needs to handle it differently from before (because it's different).